lib/hanami/view/application_context.rb

```diff
@@ -45,6 +81,11 @@ def flash
           response.flash
         end
 
+        def assets
+          @assets or
+            raise Hanami::View::MissingProviderError.new("hanami-assets")
```

Comment:
I'm not fully sold on the name of this error class, but I'm good for us to leave it as is and we can reconsider it once we move all this integration stuff into hanami itself (a better name might present itself there once we're seeing it in the context of all the integrations together).